simconnect-sdk/src/simconnect/events.rs

```diff
@@ -132,4 +132,22 @@ impl SimConnect {
 
         Ok(())
     }
+
+    /// Request that the Microsoft Flight Simulator server transmit to all SimConnect clients the specified client event.
+    #[tracing::instrument(
+        name = "SimConnect::transmit_client_event",
+        level = "debug",
+        skip(self)
+    )]
+    pub fn transmit_client_event(
+        &mut self,
+        object_id: u32,
+        event_id: ClientEventRequest,
+        dword: u32,
```

Comment:
It will be more user-friendly to take `ClientEvent` as a parameter here and use the existing `TryFrom` implementation to split it inside the function into a `ClientEventRequest` and a numeric value.